http/tokenV2/middleware.go

```diff
@@ -0,0 +1,108 @@
+package tokenV2
+
+import (
+	"fmt"
+	"strings"
+
+  	"github.com/nuts-foundation/nuts-node/http/log"
+
+        "github.com/labstack/echo/v4"
+        "github.com/lestrrat-go/jwx/jwk"
+        //"github.com/lestrrat-go/jwx/jws"
+        "github.com/lestrrat-go/jwx/jwt"
+)
+
+// New returns a new token authenticator middleware given the contents of an SSH
+// authorized_keys file. Requests containing a JWT Bearer token signed by one of
+// the specified keys will be authorized, and those not will receive an HTTP 401
+// error.
+func New(authorizedKeys []byte) (Middleware, error) {
+	// Parse the authorized keys, returning an error if it fails
+	parsed, err := parseAuthorizedKeys(authorizedKeys)
```

Comment:
Do you mean what we intend or the actual behaviour? I'm not sure I understand.

In any case I'll make sure an empty file case is tested. In terms of what we want it to do, I'm not sure an empty file *should* cause an error, as a sysadmin may clear it in an emergency to just block everybody.

Reply:
Discussion: it's misconfiguration, but hard fail is too harsh, since it might be provisioned and thus be empty at some point. Then you don't want the Nuts node to fail to start. It's worth a warn log, though.